.deployment.config.json

```diff
@@ -3,17 +3,27 @@
   "team_name": "DX",
   "general": {
     "team_jenkins": "searchuibuilds",
-    "notifications": {"slack_channels": ["#searchuibuilds"]},
+    "notifications": {
+      "slack_channels": ["#searchuibuilds"]
+    },
     "environments_order": {
       "sequential": ["dev", "qa", "prd"]
     },
-    "qa": {"start_environment_automatically": true},
+    "qa": {
+      "start_environment_automatically": true
+    },
+    "prd": {
+      "start_environment_automatically": true
```

Comment:
I know this looks bad, but hear me out 😂 

Now that we're running the security analysis before starting the deployment pipeline, I'd argue this check is not necessary anymore, because the only one thing left to check after we started the deployment pipeline is the `package_rollout`, and I reckon we do not need to double-check it.

TL;DR: The 'deploy to prod' manual decision would become the "start the pipeline" manual action

Reply:
Yes, that's perfectly fine.